docs/utilities/oredictionary.md

```diff
@@ -0,0 +1,77 @@
+OreDictionary
+======
+
+The OreDictionary exists primarily for inter-mod compatibility. 
+
+By registering ores to the OreDictionary, mods make it possible to use the ores and ingots from another, similar mod in their recipes.
+
+Despite its name, the OreDictionary is used for much more than ores. Any item that is similar to another item (for example, dyes) can be registered to and used with the OreDictionary.
+
+## OreDictionary Name Convention
+
+!!! note
+    Because OreDictionary names are meant to be shared between items from different mods, they should be fairly general. Use a name that other mods are likely to use, based on these conventions.
+	
+All OreDictionary names should be camelCase (compound words that begin with a lowercase letter, where each successive word begins with a capital letter). Avoid spaces or underscores.
```

Comment:
Forge does not impose a standard on oredict names. You can suggest this as a standard for convenience, and mention that it is a popular standard, but it is not enforced or required.